parm/config/config.resources

```diff
@@ -47,16 +48,16 @@ if [ $step = "prep" -o $step = "prepbufr" ]; then
 
 elif [ $step = "waveinit" ]; then
 
-    export wtime_waveinit="00:10:00"
+    export wtime_waveinit="00:15:00"
     export npe_waveinit=12
     export nth_waveinit=1
     export npe_node_waveinit=$(echo "$npe_node_max / $nth_waveinit" | bc)
     export NTASKS=${npe_waveinit}
 
 elif [ $step = "waveprep" ]; then
 
-    export wtime_waveprep="00:30:00"
-    export npe_waveprep=65
+    export wtime_waveprep="00:15:00"
```

Comment:
Hmm... so this is where we have different needs for a GFSv16 wave configuration versus s2s, and actually for s2s this could be 1.

Reply:
Will revert back to develop settings, thanks!